test/libyul/ObjectCompilerTest.cpp

```diff
@@ -43,7 +45,16 @@ ObjectCompilerTest::ObjectCompilerTest(string const& _filename):
 	TestCase(_filename)
 {
 	m_source = m_reader.source();
-	m_optimize = m_reader.boolSetting("optimize", false);
+	m_optimisationLevel = m_reader.enumSetting<OptimisationLevel>(
+		"optimizationLevel",
```

Comment:
Note how the setting is called `optimizationLevel` while the variable is `m_optimisationLevel`. We have a weird conflict where `OptimiserSettings.h` mainly uses `s` while CLI or test settings mostly use `z`.

I did not want to replace `optimize` with `optimisationLevel` so I ended up doing this. Not sure if it's the best solution.

Reply:
I'm fine with this but I'm also fine with just using `z` for new code.

Reply:
Well, the thing that's bugging me the most is that with `z` we whould have `OptimiserSettings` and `OptimizationLevel` (or `OptimizationPreset`) in the same file.